src/auth/auth-token.entity.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from '../users/user.entity';
+
+@Entity()
+export class AuthToken {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true })
+  keyId: string;
+
+  @ManyToOne((_) => User, (user) => user.authTokens)
+  user: User;
+
+  @Column()
+  identifier: string;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @Column({ unique: true })
+  accessTokenHash: string;
+
+  @Column({
+    nullable: true,
+  })
+  validUntil: number;
+
+  @Column({
+    nullable: true,
+  })
+  lastUsed: number;
+
+  public static create(
+    user: User,
+    identifier: string,
+    keyId: string,
+    accessToken: string,
+    validUntil?: number,
+  ): Pick<AuthToken, 'user' | 'accessTokenHash'> {
```

Comment:
That type looks wrong, it would only contain the `user` and `accessTokenHash` keys, but you set more properties.

Reply:
fixed